src/core/tests/CMakeLists.txt

```diff
@@ -2,10 +2,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+remove_definitions(-DIN_NGRAPH_LIBRARY)
```

Comment:
but better to keep this definition here and not use global headers in tests (the same for opsets)